support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -222,8 +244,92 @@ export function PaymentTsAndCs({
 						'SupporterPlus',
 						promotion,
 					)}
-				{inSupport && copyBelowThreshold(contributionType)}
+				{inSupport && copyBelowThreshold(contributionType, countryGroupId)}
 			</FinePrint>
 		</div>
 	);
 }
+
+export function SummaryTsAndCs({
+	contributionType,
+	countryGroupId,
+	currency,
+	amount,
+	amountIsAboveThreshold,
+	productNameAboveThreshold,
```

Comment:
Using `productKey` would make this less brittle to future label changes.

Reply:
Have used `productKey` throughout this file now, unpacking the associated name at the end, agree more robust that way.